packages/nocodb/tests/unit/rest/tests/meta-apis/fields.test.ts

```
@@ -0,0 +1,97 @@
+    let context: Awaited<ReturnType<typeof init>>;
+    let base: Base;
+    let table: Model;
+
+    before(async function () {
+      context = await init();
+
+      base = await createProject(context);
+      table = await createTable(context, base);
+    });
+
+    it.only(`Create basic columns ${API_VERSION}`, async function () {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)